compiler/src/iree/compiler/Codegen/LLVMGPU/Utils/LLVMGPULayoutAnalysisAndDistribution.cpp

```diff
@@ -785,86 +796,87 @@ static void distributeReductionBroadcastTranspose(
   }
 
   bodyType loopBody = [&](std::array<int, DimType::NumDims> &state) {
+    Value vector = simdToSimtMap.at(source);
+    VectorType vectorType = vector.getType().cast<VectorType>();
+    Type elementType = vectorType.getElementType();
+    int bitWidth = elementType.getIntOrFloatBitWidth();
+    bool isFP32 = bitWidth == 32;
+    uint32_t size{32};
```

Comment:
Is this meant to be the warp size? Maybe make it a static constexpr somewhere so that it is easier to generalize later?